### PR TITLE
Add support for RTL languages - VPN-3146

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

File: translations/strings.yaml [hunk 1]
```diff
@@ -690,6 +690,9 @@ inAppMessaging:
     comment: A badge shown in a message signifying that this message contains a survey
 
 global:
+  rtl:
+    value: ltr
+    comment: Write 'ltr' (left-to-right) or 'rtl' (right-to-left) to inform the app about the layout direction to use.
```
Comment (flodolo) [inline translations/strings.yaml:695]: Based on experience with Firefox, this is very brittle.

Can we check if QT provides info on the locale and its text direction? That's information included in intl libraries like ICU.

**Review by flodolo — APPROVED** [review]
Thanks, this works for me.

The only edge case is if the locale is not supported in CLDR, but I doubt we'll have this problem any time soon (RTL locales are few, and all major ones are in CLDR).

**Review by lesleyjanenorton — APPROVED** [review]
nice!

File: src/localizer.cpp [hunk 2]
```diff
@@ -5,6 +5,8 @@
 #include "localizer.h"
 #include "collator.h"
 #include "constants.h"
+#include "inspector/inspectorhandler.h"
+#include "l18nstrings.h"
```
[the remark this answers is not in the document]
Comment (bakulf) [reply src/localizer.cpp:9]: Right!

**Pull request merged**